src/Type/ItemPrice.php

```diff
@@ -177,16 +181,17 @@ public function total()
      * Retrieves the total tax amount considering all item taxes, or just the given tax
      *
      * @param TaxPrice $tax A specific tax price whose tax to calculate for this item (optional, default null)
+     * @param string $type The type of tax for which to retrieve amounts (inclusive, exclusive, or inclusive_calculated)
```

Comment:
might be better to reference `TaxPrice::INCLUSIVE`, `TaxPrice::EXCLUSIVE`, etc. rather than their string value. Wish PHP had real enums 😞

Reply:
Fair enough, I'll update